canonical-web-and-design/ubuntu.com
# Update /smart-displays page

## Done

**Waiting on the final +1 from marketing**
- Rename /digital-signage to /smart-displays
- Add redirection for /digital-signage
- Update /smart-displays page as per [copydoc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.oex3gsvjp5pm)
- Update other references (links, thank-you topics, etc) from digital-signage to smart-displays

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11368.demos.haus/iot/smart-displays
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against [copydoc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.oex3gsvjp5pm)
- Check that the redirects work ex. https://ubuntu-com-11368.demos.haus/iot/digital-signage more can be seen in the redirects.yaml changes

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4891
